addons/voice/org.openhab.voice.pollytts/src/main/java/org/openhab/voice/pollytts/internal/cloudapi/PollyClientConfig.java

```diff
@@ -0,0 +1,158 @@
+
+    }
+
+    /**
+     * save the user unique accessKey for the service
+     */
+    public void setAccessKey(String key) {
+        PollyClientConfig.accessKey = key;
+    }
+
+    /**
+     * save the user unique secretKey for the service
+     */
+    public void setSecretKey(String key) {
+        PollyClientConfig.secretKey = key;
+    }
+
+    /**
+     * save regional location for the server to be used
+     */
+    public void setRegionVal(String val) {
+        PollyClientConfig.regionVal = val;
+    }
+
+    /**
+     * save the life time for cache files
+     * 0 means forever
+     */
+    public void setExpireDate(int days) {
+        PollyClientConfig.expireDate = days;
+    }
+
+    /**
+     * get the life time for cache files
+     */
+    static public int getExpireDate() {
+        return PollyClientConfig.expireDate;
+    }
+
+    /**
+     * sets audio format specified for audio
+     */
+    public void setAudioFormat(String format) {
+        PollyClientConfig.audioFormat = format;
+    }
+
+    /**
+     * returns audio format specified for audio
+     */
+    public static String getAudioFormat() {
+        return PollyClientConfig.audioFormat;
+    }
+
+    /**
+     * get the date when cache was cleaned last
+     */
+    public static long getlastDelete() {
+        return PollyClientConfig.today;
+    }
+
+    /**
+     * set the date when cache was cleaned last
+     */
+    public static void setLastDelete(long today) {
+        PollyClientConfig.today = today;
+        ;
+    }
+
+    /**
+     * Initializes the amazon service Credentials
+     * as a one time event, saved for future reference
+     */
+    public boolean initPollyServiceInterface() {
+        // service interface not created
+        boolean initialized = false;
+        try {
+            AWSCredentials credentials = new BasicAWSCredentials(PollyClientConfig.accessKey,
+                    PollyClientConfig.secretKey);
+
+            pollyClientInterface = (AmazonPollyClient) AmazonPollyClientBuilder.standard().withRegion(regionVal)
+                    .withCredentials(new AWSStaticCredentialsProvider(credentials)).build();
+
+            DescribeVoicesRequest describeVoicesRequest = new DescribeVoicesRequest();
+            // ask Amazon Polly to describe available TTS voices.
+            DescribeVoicesResult describeVoicesResult = pollyClientInterface.describeVoices(describeVoicesRequest);
+            pollyVoices = describeVoicesResult.getVoices();
+
+            // create voice to ID translation for service invocation
+            for (Voice voice : PollyClientConfig.pollyVoices) {
+                labelToID.put(voice.getName(), voice.getId());
+            }
+
+            // Initialize expired check date to 172800000 +1
+            // run today and ~ every 2 days if cache cleaner enabled
+            // 1 day = 24 * 60 * 60 * 1000 = 86,400,000
+            today = 172800001;
+
+            initialized = true;
```

Comment:
There's no need for the initialized variable if you just `return true;` here and `return false;` in the catch.